tests/test_sparkml_pipeline.py

```diff
@@ -29,7 +29,7 @@
 
 class TestSparkMLPipeline(unittest.TestCase):
     @unittest.skipIf(not sparkml_installed(), reason="Spark-ML test requires pyspark")
-    @unittest.skipIf(LooseVersion(torch.__version__) < LooseVersion("1.6.0"), reason="Spark-ML test requires torch >= 1.16.0")
```

Comment:
Is this a copy-paste err?

Reply:
No no I fixed a typo here.

Reply:
Ok but it is confusing that you are fixing a SparkML comment in a PR related to SGD. Can you please put that in a different PR?